src/Libraries/CoreNodeModels/Selection.cs

```diff
@@ -102,8 +102,12 @@ public virtual string SelectionSuggestion
 
             this.selectionType = selectionType;
             this.selectionObjectType = selectionObjectType;
-
-            OutPortData.Add(new PortData("Elements", Resources.SelectionPortDataResultToolTip));
+            if (selectionMessage.Equals("Select Model Element"))
+            {
+                OutPortData.Add(new PortData("Element", Resources.SelectionPortDataResultToolTip));
+            }
+            else 
+                OutPortData.Add(new PortData("Elements", Resources.SelectionPortDataResultToolTip));
```

Comment:
Why this string isn't localized? 

I'd suggest to rewrite it in the other way:
```
string portName = condition ? "Element" : "Elements";
OutPortData.Add(new PortData(portName), Resources.SelectionPortDataResultToolTip));
```